packages/financial-templates-lib/src/helpers/GasEstimator.ts

```diff
@@ -192,8 +192,12 @@ export class GasEstimator {
     ]);
 
     if (this.type == NetworkType.London) {
-      this.latestMaxFeePerGasGwei = (gasInfo as LondonGasData).maxFeePerGas;
       this.latestMaxPriorityFeePerGasGwei = (gasInfo as LondonGasData).maxPriorityFeePerGas;
+      // If we are using a hardcoded maxPriorityFeePerGas value, ensure that maxFeePerGas is not set below it.
+      this.latestMaxFeePerGasGwei = Math.max(
```

Comment:
If i.maxFeePerGas is less than i.maxPriorityFeePerGas, reducing maxPriorityFeePerGas could save gas. However, your method might lead to quicker transactions.

```
this.latestMaxFeePerGasGwei = (gasInfo as LondonGasData).maxFeePerGas;
this.latestMaxPriorityFeePerGasGwei = Math.min((gasInfo as LondonGasData).maxPriorityFeePerGas,(gasInfo as LondonGasData).maxFeePerGas)
```

Reply:
Agree. For now we can opt on the safe side to get quicker inclusion